woocommerce/woocommerce-gateway-stripe
# feat: refresh account information button

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1875

Button to refresh the payment methods list, per latest designs:

![2021-10-06 16 31 45](https://user-images.githubusercontent.com/273592/136286466-9c422900-3f41-4bfb-b525-ab141e9e3f1a.gif)

Designs available at Igkr3nOx0UoGDlHrsJ4eux-fi-959%3A48387
![Screen Shot 2021-10-06 at 4 41 57 PM](https://user-images.githubusercontent.com/273592/136287396-19d6eff5-81a2-489f-b6b3-0d3997d7d7a6.png)

This button refreshes the "Pending activation" capability on the payment methods, when set.

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Ensure you have the UPE enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe
- Click the menu button
- Click the item to refresh the list
- You should see a `POST` request being made to `wp-json/wc/v3/wc_stripe/account/refresh`
